tests/conftest.py

```diff
@@ -648,53 +653,19 @@ def fixture_default_surrogate_model(request, kernel):
     return GaussianProcessSurrogate(kernel_or_factory=kernel)
 
 
-@pytest.fixture(name="allow_repeated_recommendations")
```

Comment:
Would prefer to remove dead code as much as possible, since our test suite is pretty bloated anyway. It seems with the new version, there is no more need for them at the moment. And we can always bring them back easily *if* we ever need them again. So unless you insist having them right now, I'd say let's keep as is. So feel free to reopen.